podpac/__init__.py

```diff
@@ -11,6 +11,27 @@
     Description
 """
 
+
+# Monkey match os.makedirs for Python 2 compatibility
+import sys
+import os
+_osmakedirs = os.makedirs
+def makedirs(name, mode=511, exist_ok=False):
```

Comment:
I saw @mlshapiro use the `exist_ok` param and really liked it. Turns out it was Python 3 only, hence the monkey patch.